CRM/PivotReport/AbstractData.php

```diff
@@ -100,7 +127,7 @@ public function rebuildData($cacheGroup, $entityName, array $params, $offset = 0
         $offset--;
       }
 
-      $pages = $this->getPaginatedResults($entityName, $apiParams, $offset, $multiValuesOffset, $page, $index);
+      $pages = $this->getPaginatedResults($this->name, $apiParams, $offset, $multiValuesOffset, $page, $index);
```

Comment:
`getPaginatedResults()` is also an instance method and it has access to `$this->name`, so you don't need to pass it as an argument